.env

```diff
@@ -11,8 +11,8 @@ REACT_APP_VX_SKIP_CVR_ELECTION_HASH_CHECK=FALSE
 REACT_APP_VX_SKIP_SCAN_ELECTION_HASH_CHECK=FALSE
 REACT_APP_VX_SKIP_BALLOT_PACKAGE_AUTHENTICATION=FALSE
 REACT_APP_VX_SKIP_CAST_VOTE_RECORDS_AUTHENTICATION=FALSE
-REACT_APP_VX_DISABLE_CVR_ORIGINAL_SNAPSHOTS=FALSE
+REACT_APP_VX_CAST_VOTE_RECORD_OPTIMIZATION_EXCLUDE_ORIGINAL_SNAPSHOTS=TRUE
+REACT_APP_VX_CAST_VOTE_RECORD_OPTIMIZATION_EXCLUDE_REDUNDANT_METADATA=TRUE
```

Comment:
As noted in the PR description:

> I'm enabling both CVR optimization feature flags by default, since we'll want them enabled for real elections for the foreseeable future as far as I can tell. Faster performance on election day feels more important than hypothetical cross-system compatibility, for now